packages/clay-charts/src/BillboardWrapper.tsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import React, {HTMLAttributes, RefObject} from 'react';
+import {bb, Chart, ChartOptions} from 'billboard.js';
+
+interface IProps extends ChartOptions {
+	elementProps?: HTMLAttributes<HTMLDivElement>;
+}
+
+class BillboardWrapper extends React.Component<IProps, {}> {
```

Comment:
I'd say the same here, would not it be better to be a hook?

Reply:
same as above